lib/Models/Cesium.ts

```diff
@@ -709,7 +709,7 @@ export default class Cesium extends GlobeOrMap {
             });
           }
         });
-      } else if (target.position !== undefined) {
+      } else if (target.position instanceof Cartesian3) {
```

Comment:
When `target` is a catalog item with `position` trait, it resulted in zoom hanging forever. This check prevents it.